libs/common/src/sql-db-connection/init.sh

```diff
@@ -1,54 +1,42 @@
 #!/bin/bash
 set -m
-
-execute_sql_command() {
-  /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "$MSSQL_SA_PASSWORD" -Q "$1" > /dev/null
-  return $?
-}
-
-get_now_unix_timestamp() {
-  echo $(date +%s)
-}
-
 /opt/mssql/bin/sqlservr &
 
-readonly time_limit_seconds=60
-readonly interval_seconds=5
-readonly start_time=$(get_now_unix_timestamp)
-
 echo "⚡ Initialising MSSQL Server"
 
-# Wait for SQL Server to start
-while ! execute_sql_command "SELECT 1;"
-do
-  current_time=$(get_now_unix_timestamp)
-  elapsed_seconds=$((current_time - start_time))
-
-  if [ $elapsed_seconds -gt $time_limit_seconds ]; then
-    echo "❌ Error: Failed to create database '$SQL_DB_NAME' - SQL Server did not start within $time_limit_seconds seconds."
-    exit 1
-  fi
+# Executes SQL query
+query() {
+  /opt/mssql-tools/bin/sqlcmd -C -S localhost -U sa -P "$MSSQL_SA_PASSWORD" -Q "$1" >/dev/null
+  return $?
+}
 
-  echo "❌ MSSQL failed to start after $elapsed_seconds seconds. Waiting for $interval_seconds more seconds."
-  sleep $interval_seconds
-done
+# Initial DB setup
+initialise() {
+  # 1. Database
+  query "CREATE DATABASE [$SQL_DB_NAME];"
+  query "USE [$SQL_DB_NAME];"
 
-echo "✅ MSSQL initialised, creating database '$SQL_DB_NAME'..."
```

Comment:
I think this logging is quite helpful when looking inside the container as the other logs aren't very descriptive. Is there a reason to remove them?

Reply:
I have kept the logging in place and added an extra line to distinguish between when the MSSQL service is running and when it has been initialised.